docs/installation-configuration.md

```diff
@@ -1293,52 +1293,157 @@ If the port is changed or intended to be changed, refer to the details below
-}
-```
+    {{% choosable language typescript %}}}
+
+    ```typescript
+    import * as pulumi from "@pulumi/pulumi";
+    import * as fortios from "@pulumiverse/fortios";
+
+    const fglobal = new fortios.system.Global("fglobal", {
+        admintimeout: 240,
+        hostname: "testhostname",
+        timezone: "33",
+        adminSport: 8443,
+        adminSshPort: 22,
+    });
+    ```
+
+    {{% /choosable %}}
+
+    {{% choosable language python %}}
+
+    ```python
+    import pulumi
+    import pulumiverse_fortios as fortios
+
+    fglobal = fortios.system.Global("fglobal",
+        admintimeout=240,
+        hostname="testhostname",
+        timezone="33",
+        admin_sport=8443,
+        admin_ssh_port=22)
+    ```
+
+    {{% /choosable %}}
+
+    {{% choosable language go %}}
+
+    ```go
+    package main
+
+    import (
+      "github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+      "github.com/pulumiverse/pulumi-fortios/sdk/go/fortios/system"
+    )
+
+    func main() {
+      pulumi.Run(func(ctx *pulumi.Context) error {
+        _, err := system.NewGlobal(ctx, "fglobal", &system.GlobalArgs{
+          Admintimeout: pulumi.Int(240),
+          Hostname:     pulumi.String("testhostname"),
+          Timezone:     pulumi.String("33"),
+          AdminSport:   pulumi.Int(8443),
+          AdminSshPort: pulumi.Int(22),
+        })
+        if err != nil {
+          return err
+        }
+        return nil
+      })
+    }
+    ```
+
+    {{% /choosable %}}
+
+    {{% choosable language csharp %}}
+
+    ```csharp
+    using System.Collections.Generic;
+    using System.Linq;
+    using Pulumi;
+    using Fortios = Pulumiverse.Fortios;
+
+    return await Deployment.RunAsync(() => 
+    {
+        var fglobal = new Fortios.System.Global("fglobal", new()
+        {
+            Admintimeout = 240,
+            Hostname = "testhostname",
+            Timezone = "33",
+            AdminSport = 8443,
+            AdminSshPort = 22,
+        });
+
+    });
+    ```
+
+    {{% /choosable %}}
+
+    {{% choosable language yaml %}}
+
+    ```yaml
+    resources:
+      fglobal:
+        type: fortios:system:Global
+        properties:
+          admintimeout: 240
+          hostname: testhostname
+          timezone: '33'
+          adminSport: 8443
+          adminSshPort: 22
+    ```
+
+    {{% /choosable %}}
+
+    {{< /chooser >}}
+
+    Then execute "Pulumi init; Pulumi plan; Pulumi apply". After a few seconds
+    or more (depending on your network situation), then press `Ctrl+C` to end
+    the apply command. If your network is okay, the `adminSport` will be
+    successfully set to `8443`.
+
+2. Change the Provider part of the configuration file `Pulumi.yaml`, add the
+   8443 port number to the hostname and separate them with a colon, as follows,
+   then re-apply Pulumi:
+
+    ```yaml
+    provider "fortios" {
+      hostname = "192.168.52.111:8443"
+      token    = "nx6nbGn8tnFddaa3Qy79jpjfsyw1"
+      insecure = "true"
+    }
```

Comment:
This doesn't look right either. Apologies if I'm missing something though -- I admit I'm completely unfamiliar with this provider, but this looks like HCL to me.